lib/server/sse_handler.dart

```diff
@@ -28,16 +28,37 @@ class SseConnection extends StreamChannelMixin<String> {
   /// Outgoing messages to the Browser client.
   final _outgoingController = StreamController<String>();
 
-  final Sink _sink;
+  Sink _sink;
+
+  /// How long to wait after a connection drops before considering it closed.
+  final Duration _keepAlive;
+
+  /// Whether the connection is in the timeout period waiting for a reconnect.
+  bool _isTimingOut = false;
```

Comment:
Done! However it's slightly weird in that after cancelling I also need to set it to `null` so I can check for nulls elsewhere (otherwise we'd probably need a getter that wraps up checking that it is either null or cancelled - which would look a lot like this boolean - but let me know if you think that would be better).